src/templates/detail/main.html

```diff
@@ -199,22 +199,21 @@
             <ul>
               {% if user.logged_in() and this.user.has_rated %}
                 <li>
-                  <a id="edit-review" class="button" data-app="{{ slug }}"
+                  <a class="write-review edit-review button" data-app="{{ slug }}"
```

Comment:
Yeah I was just making do with what was there. The way the JS is set is up, I think I will make it ```review-button``` and then have a ```data-edit-review``` atttr